tests/test_config_repository.py

```diff
@@ -194,3 +196,16 @@ def test_restore_singleton_state_hack() -> None:
     Pytest's test collection phase, which is before the tests are dunning - so it does not solve the problem.
     """
     Singleton.set_state(state)
+
+
+@mark.skipif(sys.version_info[0:2] < (3, 8), reason="only run on python 3.8+")
+def test_importlib_resource_load_zip_path(monkeypatch: Any) -> None:
+    config_source = ImportlibResourcesConfigSource(provider="foo", path="pkg://bar")
+    conf = config_source._read_config(
+        zipfile.Path(  # type: ignore
+            "tests/test_apps/config_source_test/importlib_resource_zip_test/conf.zip",
```

Comment:
the location of this file does not make sense to me.
test_apps is reserved to full application, it's not a generic directory.
`hydra/test_utils/configs` is a better place for it.